frontend/copr-frontend.spec

```diff
@@ -1,5 +1,5 @@
 %bcond_without check
-%if 0%{?fedora} < 32
+%if 0%{?fedora} < 32 && !0%{?openEuler}
```

Comment:
This condition can be dropped as well.  Just use %bcond_without unconditionally.